src/components/Utils/helpers.js

```diff
@@ -13,3 +14,10 @@ export const isProductDuplicated = (products, item) => {
   );
   return normalizedItemsDb.includes(normalizedItemInput);
 };
+
+export const isWithin24hours = (lastPurchasedDate) => {
+  const currentDate = +new Date();
```

Comment:
I don't think you need the `+` here

Reply:
@gmzjuliana that is an inner method of that property check the prototype of the response from firebase, look:
<img width="368" alt="Screen Shot 2021-01-31 at 10 20 42 AM" src="https://user-images.githubusercontent.com/54448243/106388833-4b141a00-63ae-11eb-9533-9a4e4109634a.png">

Reply:
The plus + symbol is to do an implicit cast to type "number" on that variable.